### PR TITLE
Handling specific case of spaces being disallowed

### PR DESCRIPTION
**Pull request by kwonye**
in username and site address

Addresses #3406

@oguzkocer if you don't mind, can you tackle this real quick as well? Thanks in advance!

Needs review: @oguzkocer

### CODE REVIEW

File: WordPress/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -955,6 +955,7 @@
     <string name="invalid_password_message">Password must contain at least 4 characters</string>
     <string name="invalid_username_too_short">Username must be longer than 4 characters</string>
     <string name="invalid_username_too_long">Username must be shorter than 61 characters</string>
+    <string name="invalid_username_no_spaces">Username cannot contain spaces</string>
```
Comment (maxme) [inline WordPress/src/main/res/values/strings.xml]: `s/cannot/can't/` - See "Use contractions" section in https://www.google.com/design/spec/style/writing.html#writing-capitalization-punctuation

File: WordPress/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -975,6 +976,7 @@
     <string name="email_reserved">That email address has already been used. Check your inbox for an activation email. If you don\'t activate you can try again in a few days.</string>
     <string name="blog_name_required">Enter a site address</string>
     <string name="blog_name_not_allowed">That site address isn\'t allowed</string>
+    <string name="blog_name_no_spaced_allowed">Site address cannot contain spaces</string>
```
Comment (maxme) [inline WordPress/src/main/res/values/strings.xml]: `s/cannot/can't/`

**Comment by oguzkocer** [comment]
Looks like @maxme already handled this and I might have missed that contractions issue. TIL I guess :)

**Comment by kwonye** [comment]
@maxme thanks for that reminder. Addressed

**Comment by maxme** [comment]
Looks good :shipit:

**Pull request merged**